e2e/playwright/favorite-actions/src/tests/mark-favorite.e2e.ts

```diff
@@ -0,0 +1,183 @@
+      trashcanApi = await TrashcanApi.initialize(username, username);
+      nodesApi = await NodesApi.initialize(username, username);
+      favoritesApi = await FavoritesPageApi.initialize(username, username);
+      sharedApi = await SharedLinksApi.initialize(username, username);
+
+      parentId = (await nodesApi.createFolder(parent)).entry.id;
+      fileFavUIId = (await nodesApi.createFile(fileFavUI, parentId)).entry.id;
+      fileNotFav1Id = (await nodesApi.createFile(fileNotFav1, parentId)).entry.id;
+      fileNotFav2Id = (await nodesApi.createFile(fileNotFav2, parentId)).entry.id;
+      fileNotFav3Id = (await nodesApi.createFile(fileNotFav3, parentId)).entry.id;
+      fileNotFav4Id = (await nodesApi.createFile(fileNotFav4, parentId)).entry.id;
+      fileFav1Id = (await nodesApi.createFile(fileFav1, parentId)).entry.id;
+      fileFav2Id = (await nodesApi.createFile(fileFav2, parentId)).entry.id;
+      fileFav3Id = (await nodesApi.createFile(fileFav3, parentId)).entry.id;
+      fileFav4Id = (await nodesApi.createFile(fileFav4, parentId)).entry.id;
+      folderId = (await nodesApi.createFolder(folder, parentId)).entry.id;
+      await nodesApi.createFile(fileNotFavUI, parentId);
+
+      const currentFavoritesTotalItems = await favoritesApi.getFavoritesTotalItems(username);
+      await favoritesApi.addFavoritesByIds('file', [fileFavUIId, fileFav1Id, fileFav2Id, fileFav3Id, fileFav4Id]);
+      await favoritesApi.waitForApi(username, { expect: currentFavoritesTotalItems + 5 });
+
+      await sharedApi.shareFilesByIds([fileFav1Id, fileFav2Id, fileFav3Id, fileFav4Id]);
+      await sharedApi.shareFilesByIds([fileNotFav1Id, fileNotFav2Id, fileNotFav3Id, fileNotFav4Id]);
+      await sharedApi.waitForFilesToBeShared([
+        fileFav1Id,
+        fileFav2Id,
+        fileFav3Id,
+        fileFav4Id,
+        fileNotFav1Id,
+        fileNotFav2Id,
+        fileNotFav3Id,
+        fileNotFav4Id
+      ]);
+    } catch (error) {
+      console.error(`beforeAll failed: ${error}`);
+    }
+  });
+
+  test.beforeEach(async ({ loginPage }) => {
+    await Utils.tryLoginUser(loginPage, username, username, 'beforeEach failed');
+  });
+
+  test.afterAll(async () => {
+    await Utils.deleteNodesSitesEmptyTrashcan(nodesApi, trashcanApi, 'afterAll failed');
+  });
+
+  test.describe('on Personal Files', () => {
+    test.afterAll(async () => {
+      try {
+        await favoritesApi.addFavoritesByIds('file', [fileFavUIId, fileFav1Id, fileFav2Id, fileFav3Id, fileFav4Id]);
+        await favoritesApi.addFavoriteById('folder', folderId);
+        await favoritesApi.removeFavoritesByIds(username, [fileNotFav1Id, fileNotFav2Id, fileNotFav3Id, fileNotFav4Id]);
+      } catch (error) {}
+    });
+
+    test.beforeEach(async ({ personalFiles }) => {
+      await personalFiles.dataTable.performClickFolderOrFileToOpen(parent);
+    });
+
+    test('Favorite action has empty star icon for an item not marked as favorite', async ({ personalFiles }) => {
```

Comment:
can you please add test add from test rail..

 https://hyland.atlassian.net/projects/XAT?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.xpandit.plugins.xray__testing-board&ac.selectedFolder=66311decae42b5c04ceb3fc1&ac.projectId=14544#!page=test-repository&selectedFolder=66311decae42b5c04ceb3fc1

Reply:
Added XRay IDs if that's what you meant :D